Fluent/Controls/Backstage.cs

```diff
@@ -378,6 +398,33 @@ private void CreateAndAttachBackstageAdorner()
 
             var layer = AdornerLayer.GetAdornerLayer(this);
             layer.Add(this.adorner);
+
+            layer.CommandBindings.Add(new CommandBinding(RibbonCommands.OpenBackstage,

```

Comment:
This does not seem to be related to your fix. Why are you adding a new CommandBinding to the layer? The Backstage itself already has such a CommanBinding.